src/backend/distributed/utils/metadata_cache.c

```diff
@@ -339,10 +365,12 @@ CitusHasBeenLoaded(void)
 Oid
 DistShardRelationId(void)
 {
-	static Oid cachedOid = InvalidOid;
+	Oid cachedOid = metadataCacheState.distShardRelationId;
```

Comment:
The usual pattern in postgres/citus is to use static variables (e.g. with a common prefix). We'd like our code to use consistent patterns when possible. It also saves you from requiring separate declaration and initialisation at the top.